src/objects.js

```diff
@@ -1,41 +1,85 @@
 const createPerson = (name, age) => {
   // your code here
+  return person = {
+    name:name,
+    age:age,
+  };
 };
 
 const getName = object => {
   // your code here
+  return object.name;
 };
 
 const getProperty = (property, object) => {
   // your code here
+  return object[property];
 };
 
 const hasProperty = (property, object) => {
   // your code here
+  if (object.hasOwnProperty(property) == true) {
+    return true;
+  } else {
+    return false;
+  }
 };
 
 const isOver65 = person => {
   // your code here
+  if (person.age > 65) {
+    return true; 
+  } else {
+    return false;
+  }
 };
 
 const getAges = people => {
   // your code here
+  let arrAges = []
```

Comment:
arrAges is never reassigned, use const instead. I can see that it can look confusing when you are modifying an array, but modifying and reassigning is not the same thing. You can modify an array (or an object)which has been declared with const.